### PR TITLE
fix: update nav

### PR DESCRIPTION
**Pull request by vinckr**
As discussed with @BlueAcidFrog @tacurran ,
preparation for `/product #171`

### CODE REVIEW

File: src/page-content/navigation/navigation-header.tsx [hunk 1]
```diff
@@ -40,36 +40,36 @@ export const dropdownMenu: Pick<NavProps, 'dropdownMenu'> = {
       ]
     },
     {
-      title: 'Ory Open Source',
+      title: 'Solutions',
       mainMenu: [
         {
-          title: 'Overview',
-          description: "Ory's heart is Open Source",
+          title: 'Ory is Open Source',
```
[the remark this answers is not in the document]
Comment (BlueAcidFrog) [reply src/page-content/navigation/navigation-header.tsx:46]: I would rename it as Overview to put the focus more on the product and start moving OS into the Developer nav.

File: src/page-content/navigation/navigation-header.tsx [hunk 2]
```diff
@@ -40,36 +40,36 @@ export const dropdownMenu: Pick<NavProps, 'dropdownMenu'> = {
...
-          title: 'Oathkeeper',
-          description: 'Zero Trust Networking',
+          title: 'Zero Trust Networking',
+          description: 'Ory Oathkeeper',
           to: '/oathkeeper'
         },
         {
-          title: 'GitHub',
-          description: 'Participate in Development',
+          title: 'Community Driven Development',
```
Comment (BlueAcidFrog) [inline src/page-content/navigation/navigation-header.tsx:71]:
```suggestion
          title: 'Community Driven Development',
```

Comment (BlueAcidFrog) [inline src/page-content/navigation/navigation-header.tsx:71]: can we move this into Dev section as well?

Comment (vinckr) [reply src/page-content/navigation/navigation-header.tsx:71]: I would leave it there, but let's wait for a review.

**Pull request closed without merging**